doc/next/_sources/reference/get-feed-import-id-detail.rst.txt

```diff
@@ -1,9 +1,84 @@
 .. index:: GET /feed/import/{id}/detail
+
 .. _get_feed_import_id_detail:
 
-GET /feed/import/{id}/detail
+GET /feed/import/{id}/detail V2
+============================
```

Comment:
does this not throw warnings, that the line is not long enough?

Reply:
made it longer